inbc-program/inbc/parser/source_app_parser.py

```diff
@@ -19,15 +27,16 @@ def application_add(args: argparse.Namespace) -> str:
     manifest = ('<?xml version="1.0" encoding="utf-8"?>' +
                 '<manifest>' +
                 '<type>source</type>' +
-                '<applicationSource>' +
-                '<add><gpg>'
-                '{0}' +
-                '{1}'
-                '</gpg><repo><repos>').format(create_xml_tag(arguments, "uri"),
-                                              create_xml_tag(arguments, "keyname"))
+                '<applicationSource>' + '<add>')
```

Comment:
**Suggestion:** Replace the manual XML header creation with a constant to avoid repetition. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    XML_HEADER = '<?xml version="1.0" encoding="utf-8"?>'
    manifest = (f'{XML_HEADER}'
                '<manifest><type>source</type>'
                '<applicationSource><add>')
```